frontend/src/App.css

```diff
@@ -1,6 +1,13 @@
+.Swiper {
+  text-align: center;
+  height: 50vh;
+  width: 100%;
+  margin-bottom: 10%;
+ }
+
 .map {
   background-color: rgba(228, 228, 228, 0.486);
   padding: 1rem;
   border-radius: 3%;
   width: fit-content;
-}
+ }
```

Comment:
Hmm ? 🧐 
How did you add a single blank space here ? Prettier should have caught/fixed that